caas/kubernetes/provider/application/scale_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application_test
+
+import (
+	"context"
+
+	"github.com/juju/errors"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/juju/juju/caas"
+)
+
+func (s *applicationSuite) TestApplicationScaleStateful(c *gc.C) {
```

Comment:
May as well add a test for deployment too.

Reply:
can do